secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala

```diff
@@ -145,7 +147,7 @@ class FeedbackContext[CT,CV](user: String,
           if (filtered.nonEmpty) {
             val newRow = Map(
               cookie.systemId -> rowId
-            ) ++ filtered.map { case (id, value) => (id, value)}.toMap
+            ) ++ filtered.map { case (id, value) => (id, value) }.toMap
```

Comment:
Huh.  You didn't introduce this, but I'm having a hard time seeing what that `.map` call is supposed to do.